### PR TITLE
Add SSL expirtaion alert policy

### PR DESCRIPTION
**Pull request by Monska85**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -74,3 +74,40 @@ resource "google_monitoring_uptime_check_config" "https_uptime" {
...
+    }
+    display_name = "SSL certificate expiring soon (${each.value} days)"
+  }
+
+  user_labels = {
+    version = "1"
+    uptime  = "ssl_cert_expiration"
+  }
+
+  notification_channels = var.alert_notification_channels
```
Comment (paolomainardi) [inline main.tf:112]: is this a known variable ?

File: main.tf [hunk 2]
```diff
@@ -74,3 +74,40 @@ resource "google_monitoring_uptime_check_config" "https_uptime" {
...
+        ]
+      }
+    }
+    display_name = "SSL certificate expiring soon (${each.value} days)"
+  }
+
+  user_labels = {
+    version = "1"
+    uptime  = "ssl_cert_expiration"
+  }
```
Comment (paolomainardi) [inline main.tf:110]: What are `user_labels`?

File: main.tf (hunk 1 above)
Comment (Monska85) [reply main.tf:112]: Yes, it this the same variable used for the `failure_alert` for the http checks ([used here](https://github.com/sparkfabrik/terraform-sparkfabrik-gcp-http-monitoring/blob/main/main.tf#L34)).

File: main.tf (hunk 2 above)
Comment (Monska85) [reply main.tf]: I have fixed this using variable.

**Pull request merged**